extra/1-currency-conversion.js

```diff
@@ -15,8 +17,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
-
+function convertToBRL(price) {
+  let newPrice = price * 0.99 * 5.7;
```

Comment:
This solution looks good to me - I like your use of multiplication by `0.99` to reduce the original value by 1%.
Think about this function for a moment from the perspective of somebody who perhaps is a bit less familiar.
They probably won't have much idea of what the `5.7` means.
Could you assign 5.7 to a variable with a clear name showing what it represents ?